src/main/scala/cromwell/Main.scala

```diff
@@ -19,7 +20,20 @@ object Actions extends Enumeration {
 
 object Main extends App {
 
-  getAction(args.headOption map { _.capitalize }) match {
+  val props = System.getProperties
+  val loggerVar = props.getProperty("CROMWELL_LOGGER")
+  if (loggerVar != null && loggerVar.length == 0) {
```

Comment:
You can say something like `Option(props.getProperty("CROMWELL_LOGGER")).contains(_.length == 0)` here